tests/testdata/ramendr.openshift.io_drclusters.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     listKind: DRClusterList
     plural: drclusters
     singular: drcluster
-  scope: Namespaced
+  scope: Cluster
```

Comment:
Good catch for this ! Can you check if this CRD matches with what the Ramen has currently ?

Reply:
yes, this matches ramen's CRD